cpp/src/IceGrid/Parser.cpp

```diff
@@ -2516,24 +2516,24 @@ Parser::invalidCommand(const list<string>& s)
     string cat = *s.begin();
     if (_helpCommands.find(cat) == _helpCommands.end())
     {
-        consoleErr << "unknown `" << cat << "' command (see `help' for more info)" << endl;
+        consoleErr << "unknown '" << cat << "' command (see `help' for more info)" << endl;
```

Comment:
One slipped through here:
```suggestion
        consoleErr << "unknown '" << cat << "' command (see 'help' for more info)" << endl;
```

Reply:
Fixed.